Makefile

```diff
@@ -0,0 +1,21 @@
+TARGET_EXEC:=zelda3
+ROM:=tables/zelda3.sfc
+SRCS:=$(wildcard *.c snes/*.c)
+OBJS:=$(SRCS:%.c=%.o)
+GEN:=$(shell grep -hor tables/generated.*.h --include \*.c)
```

Comment:
Should the grep command have a `.` at the end?
`$(shell grep -hor tables/generated.*.h --include \*.c .)`

Make is hanging after `grep: warning: recursive search of stdin` on my machine.

Reply:
(maybe this is a unix vs. mac thing). my grep version is `grep (BSD grep) 2.5.1-FreeBSD`